### ISSUE TITLE
Restructuring Directory

### ISSUE THREAD

**Issue by leopripos** [sebas77/Svelto.Tasks#9]
Hello @sebas77,

What do you think about restructuring the directory so we can use Svelto.Tasks as submodule (git submodule) like Svelto.ECS.

My idea is moving all file from ./Assets/Scripts/Svelto/TaskRunner to root of repository.

**Comment by sebas77** [comment]
I thought several times about it, but so far I prefer to keep tasks indipendent by the official repository so that I can update it without needing to test svelto.ecs every time as well. I anyway update svelto tasks every time commit in svelto ecs

**Comment by leopripos** [comment]
Sorry, what i mean is not about the dependency of Task and ECS code. I mean just directory structure of task. I am using ECS just as the example because its structure already good enough.

Currently, the the root directory like this:
<img width="987" alt="screen shot 2017-11-26 at 17 07 58" src="https://user-images.githubusercontent.com/11954082/33239050-8f9b2bc6-d2cc-11e7-8733-c807bc76d925.png">

What do you think make the root directory like this:
<img width="984" alt="screen shot 2017-11-26 at 17 10 11" src="https://user-images.githubusercontent.com/11954082/33239069-f2101a32-d2cc-11e7-9699-33259c9be15d.png">

Basically, it just :
1. Move current all file inside `./Assets/Scripts/Svelto/TaskRunner` file to root directory
2. Remove other file (Assets folder and Assets/*/*) or move to other repository
3. Add change git ignore to ignore meta file (unity meta file)

**Comment by sebas77** [comment]
Sorry my bad I thought it was the ECS repository. You basically are suggesting to split the unit tests / examples from the code. I'll think about it.

**Comment by leopripos** [comment]
Yes, because i want to make it as submodule (git submodule) of my project.

**Comment by sebas77** [comment]
done
